gui/run.py

```diff
@@ -0,0 +1,12 @@
+import os
+import sys
+
+from cryptography.fernet import Fernet
+from streamlit.web import cli as stcli
+
+if __name__ == "__main__":
+    st_server_base_url_key = Fernet.generate_key()
+    os.environ["STREAMLIT_SERVER_BASE_URL_PATH"] = st_server_base_url_key.decode("utf-8")
```

Comment:
I think it's better to backup original setting if it existed.

Reply:
fixed at 965c2a2740f767a18caa6f02fa138baced0fb41b